lib/sage.ex

```diff
@@ -279,26 +290,28 @@ defmodule Sage do
   end
 
   @doc """
-  Registers tracer for a Sage execution.
+  Registers a tracer for a Sage execution.
 
   Registering duplicated tracing callback is not allowed and would raise an
   `Sage.DuplicateTracerError` exception.
 
   All errors during execution of a tracing callbacks would be logged,
   but it won't affect Sage execution.
 
-  Tracing module must implement `Sage.Tracer` behaviour.
+  Tracer can be a module that must implement `Sage.Tracer` behaviour,
+  a function, or a tuple in a shape of `{module, function, [arguments]}`.
```

Comment:
```suggestion
  a function, or a tuple in a shape of `{module, function, [extra_arguments]}`. 
  
  In any case, the function called should follow the definition of `c:Sage.Tracer.handle_event/3`
  and accept at least 3 required arguments that are documented by the callback.
```

Not sure if this is the best way to describe arguments of a function, but we definitely need to leave a note on it.

Reply:
Way better than I described ❤️